### ISSUE TITLE
Please add ability view tabs-panel on no javascript

### ISSUE THREAD

**Issue by soulcreate** [zurb/foundation-sites#8530]
Please add this to scss
```
html.no-js{
	.tabs-panel{
		display: block;
	}
}
```
to display .tabs-panel when javascript not supporting by device

**Comment by andycochran** [comment]
Relatedly, I don't think there's anything in the docs that recommends using `<html class="no-js">`. Should be added to the [JavaScript](http://foundation.zurb.com/sites/docs/javascript.html) page?

**Comment by kball** [comment]
@soulcreate seems reasonable, would you like to submit a PR for this?

@andycochran yes, probably should.  @rafibomb can you take a look?

**Comment by DanielRuf** [comment]
Bump.

This + accordion probably and also for the print mode might make sense.

**Comment by DanielRuf** [comment]
@ncoden @kball what can we do to bring this issue forward?

**Comment by ncoden** [comment]
For now there is no "no javascript" mode for Foundation. I could make sense to add it but it should be consistent across components.

**Comment by DanielRuf** [comment]
Definitely. So it would be some sort of new feature for all components (no-js mode / styles).

**Comment by DanielRuf** [comment]
We will revisit this for 7 then as this will be a new feature.
